release.nix

```diff
@@ -17,7 +17,7 @@ let
       releaseTools.sourceTarball {
         name = "nix-tarball";
         version = builtins.readFile ./version;
-        versionSuffix = if officialRelease then "" else "pre${toString nix.revCount}_${nix.shortRev}";
+        versionSuffix = if officialRelease then "" else "pre${toString nix.revCount}.${nix.shortRev}";
```

Comment:
The convention I've seen used for this in other Debian and Ubuntu packages is `<normal_version_number>+git<little endian date>.<time>.<hash-prefix>`, e.g., [`0.2.0+git20170602.1830.1cb3fb`](https://launchpad.net/~oibaf/+archive/ubuntu/graphics-drivers/+packages).

Reply:
We sadly don't have date and time information here and I don't want to change the naming schema much (we are distribution-agnostic after all, or is this more common than Debian derivatives?).

Reply:
Why is the version format changed?

Reply:
This is because Debian explicitly forbids usage of underscores in versions:
```
dpkg-buildpackage: warning:     debian/changelog(l1): version '2.0pre0_0000000' is invalid: version number contains illegal character '_'
```